actions/utils.spec.ts

```diff
@@ -33,4 +33,114 @@ describe('test against concrete examples', () => {
     it('should fail with no selectors', () => {
         expect(isComposableCowCompatible('0xdeadbeefdeadbeef')).toBe(false)
     })
-})
\ No newline at end of file
+})
+
+describe('handle nethermind reversion (simple)', () => {
+
+    const SINGLE_ORDER_NOT_AUTHED_ERROR: any = {
```

Comment:
Nitpick: declare the consts outside the test fn, unless it's something that needs the local context
Also, why the type `any`?